src/promote_literals.cpp

```diff
@@ -41,8 +41,9 @@ void promote_literals::apply(module_pass_manager& mpm) const
     {
         if(ins->name() == "@literal")
         {
-            auto new_lit = root_module->add_literal(ins->get_literal());
-            for(auto out_ins : ins->outputs())
+            auto new_lit    = root_module->add_literal(ins->get_literal());
+            auto ins_output = ins->outputs();
```

Comment:
Can you make the variable plural?

Reply:
I created a new commit with the new variable name.